lib/pxtesthandler.js

```diff
@@ -0,0 +1,44 @@
+
+
+
+module.exports = {
+    customRequestHandler: customRequestHandler}
```

Comment:
might want to export this using a different name, as it's too general. maybe `testModeRequestHandler` or something like that